src/stitching/defaultMergedResolver.ts

```diff
@@ -20,6 +20,10 @@ const defaultMergedResolver: GraphQLFieldResolver<any, any> = (parent, args, con
 
   let result = parent[responseKey];
 
+  if (result == null) {
```

Comment:
Is the double equal intended here (for type coercion), or could we change this to be a bit more strict with a `===`?

Reply:
I used it to cover `result === null || result === undefined`. ◀️ Want me to change it to that?

Reply:
No - that's fine! Just wanted to double check.